frontend/smartmeter-datacollector-configurator/src/App.vue

```diff
@@ -9,17 +9,26 @@
       </div>
     </header>
     <configurator />
+    <footer class="footer has-background-white">
+      <p class="content has-text-centered is-size-5">
+        <a href="https://github.com/scs/smartmeter-datacollector/wiki" target="_blank">Documentation</a> -
```

Comment:
Maybe also add a "Source Code" or "GitHub" link pointing to https://github.com/scs/smartmeter-datacollector.

Reply:
I added this link in the footer as well. However, the footer is still work in progress until we receive the correct text from our client.